RCNN_tf.py

```diff
@@ -23,10 +22,11 @@
 PRECISION_TF = tf.float32
 PRECISION_NP = np.float32
 PADDING_LIST = ['SAME', 'VALID']
+initializer = tf.contrib.layers.variance_scaling_initializer()
```

Comment:
Good. Maybe we could capitalize it macro style, but this is also fine!

Reply:
No, that's true. Thanks for pointing it out!